README.md

```diff
@@ -1,5 +1,84 @@
-## OCi8 for ddev
+# DDEV OCI8 Add-on
 
-```ssh
+This add-on installs Oracle Instant Client and the PHP OCI8 extension for DDEV projects, supporting both AMD64 and ARM64 architectures.
+
+## Requirements
+
+- **DDEV** v1.23+
+- **PHP** 8.2 or higher
+
+## Features
+
+- Installs Oracle Instant Client (Basic + SDK)
+- Configures PHP OCI8 extension
+- Supports both AMD64 and ARM64 architectures
+- Automatic library path configuration
```

Comment:
Since this provides nothing but client support, please say that it's intended for that usage, give an example of how it would be used.